.cascade-code/Chapman.edu/_cascade/templates/home_page_v3.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+    <head>
+        <meta charset="utf-8"/>
+        <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible"/>
+        <title>
+            <system-page-title/>
+        </title>
+        <system-page-meta-keywords/>
+        <system-page-meta-description/>
+        <system-page-meta-author/>
+        <system-region name="OG_TAGS"/>
+        <system-region name="META VIEWPORT"/>
+        <!-- Jquery, Typkit, Common Assets -->
+        <script src="//use.typekit.net/eyn5jyy.js" type="text/javascript"></script>
+        <script type="text/javascript">try{Typekit.load();}catch(e){}</script>
+        <system-region name="JQUERY"/>
+        <system-region name="CASCADE ASSETS"/>
```

Comment:
Removed `<link href="/_files/css/cu_icons.css" rel="stylesheet" type="text/css"/>`:

https://github.com/chapmanu/cascade-assets/pull/288/commits/025b2115e61c186a2b2cae24c31e8975ac9c158c

It would override fix when present. Code has been moved to `app/assets/stylesheets/components/blog-icons.scss`.